### DISCUSSION TITLE
C++ version

### DISCUSSION THREAD

**Discussion by DDATT** [p0p4k/vits2_pytorch#95] [Show and tell]
I have created a repository for implementing the VITS2 model in C++ at [VITS2-CPP](https://github.com/DDATT/Vits2-onnx-cpp). Hope it can be helpful to everyone.

**Comment by michal-bida** [comment]
Thanks for sharing! How much faster it gets? 

I am currently hosting my VITS2 as onnx running on nvidia Triton - it runs quite smooth.

**Reply by DDATT** [reply]
I haven't tested with a GPU yet. But with Intel Core i5-8400 2.8GHz x 6 CPU, on Linux env, it takes 2.5 seconds to generate a 10-second audio, which is almost similar to the Python version (2.2 seconds to generate from the same input).
